### ISSUE TITLE
System.Deployment.Application.InvalidDeploymentException, OutlookPrivacyPlugin.vsto

### ISSUE THREAD

**Issue by TheSwede86** [dejavusecurity/OutlookPrivacyPlugin#74]
What version of Windows?
Win 7 Enterprise x64

What version of Outlook (2010, 2013, x86/x64)?
Outlook 2010 x64

---

Hi!

Getting this output when trying to install "Outlook-Privacy-Plugin" on my machine:

"
Name: 
From: file:///C:/Users/<blanked>/Desktop/Outlook Privacy Plugin/OutlookPrivacyPlugin-BETA-40/OutlookPrivacyPlugin-BETA-40/OutlookPrivacyPlugin.vsto

************** Exception Text **************
System.Deployment.Application.InvalidDeploymentException: Exception reading manifest from file:///C:/Users/<blanked>/Desktop/Outlook%20Privacy%20Plugin/OutlookPrivacyPlugin-BETA-40/OutlookPrivacyPlugin-BETA-40/OutlookPrivacyPlugin.vsto: the manifest may not be valid or the file could not be opened. ---> System.Deployment.Application.InvalidDeploymentException: Manifest XML signature is not valid. ---> System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignatureReturningKey(AsymmetricAlgorithm& signingKey)
   at System.Deployment.Internal.CodeSigning.SignedCmiManifest.Verify(CmiManifestVerifyFlags verifyFlags)
   at System.Deployment.Application.Manifest.AssemblyManifest.ValidateSignature(Stream s)
   --- End of inner exception stack trace ---
   at System.Deployment.Application.Manifest.AssemblyManifest.ValidateSignature(Stream s)
   at System.Deployment.Application.ManifestReader.FromDocument(String localPath, ManifestType manifestType, Uri sourceUri)
   --- End of inner exception stack trace ---
   at Microsoft.VisualStudio.Tools.Applications.Deployment.ClickOnceAddInDeploymentManager.GetManifests(TimeSpan timeout)
   at Microsoft.VisualStudio.Tools.Applications.Deployment.ClickOnceAddInDeploymentManager.InstallAddIn()
"

I have unblocked the zip and extracted it before trying to install it, GPG4WIN is installed. The file resides on the users desktop (i.e. "C:\") so it isn't on a network share. Their computers fulfill the .NET-requirement (they have v.4.5).

Any ideas on what might cause this? Tried searching for answers but all seem to involve unblocking the zip-file and I have already done that.

My computer runs the plugin just fine but it is connected to a workgroup while the other computers (two I tried on) are connected to the domain, thinking if there is a restriction set that the domain does not allow installation of programs that have untrusted certificates. On my "workgroup"-computer I haven't installed the certificate but it still works, although there might be other restrictions since they are using a domain.

Oh and when I installed gpg4win on my computer at home the plugin for Outlooks that comes with gpg4win (GpgOL) works fine, it does not work however either on the "workgroup"-computer or the "domain"-computers at work. I can't even find it in "Add-ins" in Outlook on any machine (at work).

Best Regards - Karl, Sweden

---
Earlier post on GoogleCode:
https://code.google.com/p/outlook-privacy-plugin/issues/detail?can=2&start=0&num=100&q=&colspec=ID%20Type%20Status%20Priority%20Milestone%20Owner%20Summary&groupby=&sort=&id=148
---

**Comment by Meddington** [comment]
duplicate of #77
